packages/ndla-licenses/src/getCopyString.ts

```diff
@@ -44,6 +64,132 @@ const makeDateString = () => {
   return `${dd}.${mm}.${yyyy}`;
 };
 
+export const getDateString = (locale: string, date?: string) => {
+  const getLocale = (locale?: string) => {
+    if (locale === 'en') {
+      return 'EN-us';
+    } else if (locale === 'nn') {
+      return 'NO-nn';
+    }
+    return 'NO-nb';
+  };
```

Comment:
Vi burde finne en mer bærekraftig måte å håndtere dette på fremover. Man burde strengt tatt ikke trenge å gå gjennom kodebasene med en kam for å legge til flere locales.

Reply:
Jeg har forenklet koden litt nå, men problemet er ikke løst. `toLocaleDateString` godtar kortere språk-koder også, men ikke `nn` og `nb`, fallbacker derfor til `no`.